csharp/Platform.IO/ConsoleCancellation.cs

```diff
@@ -54,6 +82,15 @@ public void Wait()
             }
         }
 
+        /// <summary>
+        /// <para>Removes the <see cref="OnCancelKeyPress"/> handler, for the <see cref="Console.CancelKeyPress"/> event and attempts to dispose the object.</para>
+        /// <para>Удаляет обработчик <see cref="OnCancelKeyPress"/> для события <see cref="Console.CancelKeyPress"/> и выполняет попытку высвободить объект.</para>
+        /// </summary>
+        /// <param name="manual"></param>
+        /// <param name="wasDisposed">
+        /// <para>A <see cref="Boolean"/> value that determines whether the <see cref="Source"/> was disposed.</para>
```

Comment:
`A <see cref="Boolean"/> value that determines whether the <see cref="ConsoleCancellation"/> was released before the call to this method.`

ConsoleCancellation is our class/object, not the `Source`.

Reply:
It cases like this : https://github.com/linksplatform/IO/blob/4c7afc23b935d9b5111cfcb06099d7fa524a964d/csharp/Platform.IO/ConsoleCancellation.cs#L14 We use clarification after <see>; must not we have to do this with objects?
`the <see cref="Source"/>` -> `the <see cref="Source"/> object`

Reply:
Yes.